http/headers/origin.json

```diff
@@ -20,7 +20,7 @@
             },
             "firefox": {
```

Comment:
How about:

```json
             "firefox": [
               {
                 "version_added": 59
               },
               {
                 "version_added": true,
                 "partial_implementation": true,
                 "notes": "Not sent with <code>POST</code> requests until Firefox 58, see <a href='https://bugzil.la/446344'>bug 446344</a>."
               }
             ]
```

Reply:
@chrisdavidmills would this work for you?

Reply:
My suggestion misses the required pref and should also be revised as "Not sent with *same-origin* POST".

Reply:
For the 59 information, sdd the pref, and also add a note long the lines of "sent with non-CORS requests since Fx59, unless they are GET or HEAD"

Reply:
I've updated this. Does it look good to you now, @chrisdavidmills?